packaging/microshift/config.yaml

```
@@ -29,3 +29,20 @@ apiServer:
 debugging:
   # Log verbosity ('Normal', 'Debug', 'Trace', 'TraceAll'):
   #logLevel: 'Normal'
+
+etcd:
+  # The limit on the size of the etcd database, in bytes; thee etcd server will start failing writes if its size on disk reaches this value. (Default: 2GB)
```

Can we do something nicer than make this "in bytes"? Would we expect anyone to ever set this lower than 1MB, for example? I've seen some other kube-related API fields that parse strings like `1GB`. Is that code somewhere we could share it?

```suggestion
  # The limit on the size of the etcd database, in bytes; the etcd server will start failing writes if its size on disk reaches this value. (Default: 2GB)
```